src/ontology/mondo-edit.obo

```diff
@@ -481911,14 +481911,14 @@ property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/888/biliary-
 
 [Term]
 id: MONDO:0022575
-name: biliary hypoplasia
-def: "A syndromic disease characterized by a small ductal system and reduction in the number of interlobular bile ducts." [https://orcid.org/0000-0001-5208-3432, PMID:12664410]
+name: obsolete biliary hypoplasia
+def: "OBSOLETE. A syndromic disease characterized by a small ductal system and reduction in the number of interlobular bile ducts." [https://orcid.org/0000-0001-5208-3432, PMID:12664410]
 comment: Editor note: consider making this a finding; Reason of obsoletion: is a phenotype and not a disease - MONDO:excludePhenotype. Term to consider: -
 subset: obsoletion_candidate
-is_a: MONDO:0002254 {source="https://orcid.org/0000-0002-6601-2165"} ! syndromic disease
+property_value: IAO:0000231 OMO:0001000 {source="MONDO:excludeNonDisease"}
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7548" xsd:anyURI
 property_value: IAO:0006012 "2024-07-01" xsd:string
```

Comment:
@yshwetar remove this too. After the term is obsoleted, it is not "scheduled for obsoletion" anymore